group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -1701,7 +1720,6 @@ private void throwIfStreamsGroupHeartbeatRequestIsInvalid(
             }
             // TODO: check that active, standby and warmup do not intersect
         } else if (request.memberEpoch() == LEAVE_GROUP_STATIC_MEMBER_EPOCH) {
-            throwIfEmptyString(request.memberId(), "MemberId can't be empty.");
```

Comment:
Don't need this here anymore since we first check if `memberId` is null or empty before reaching the if/else statements